static/app/views/starfish/views/appStartup/screenSummary/systemApplicationBreakdown.tsx

```diff
@@ -0,0 +1,218 @@
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {prepareQueryForLandingPage} from 'sentry/views/performance/data';
+import MiniChartPanel from 'sentry/views/starfish/components/miniChartPanel';
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {formatVersionAndCenterTruncate} from 'sentry/views/starfish/utils/centerTruncate';
+import {STARFISH_CHART_INTERVAL_FIDELITY} from 'sentry/views/starfish/utils/constants';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
+import Breakdown from 'sentry/views/starfish/views/appStartup/breakdown';
+import {useTableQuery} from 'sentry/views/starfish/views/screens/screensTable';
+
+const SYSTEM_COLOR = '#D6567F';
+const APPLICATION_COLOR = '#444674';
+
+const ANDROID_APPLICATION_SPAN_OPS = [
+  'contentprovider.load',
+  'application.load',
+  'activity.load',
+];
+
+const IOS_APPLICATION_SPAN_DESCRIPTIONS = ['Initial Frame Render'];
+
+// Since we don't collect a tag that indicates whether a span is system or
+// application, we're going to use the span.op and span.description to
+// determine whether a span is system or application.
+function aggregateSystemApplicationBreakdown(data: TableDataRow[]) {
+  return data.reduce((acc, row) => {
+    const spanOp = row['span.op'] as string;
+    const spanDescription = row['span.description'] as string;
+
+    let type: 'system' | 'application' = 'system';
+    if (
+      ANDROID_APPLICATION_SPAN_OPS.includes(spanOp) ||
+      IOS_APPLICATION_SPAN_DESCRIPTIONS.includes(spanDescription)
+    ) {
+      type = 'application';
+    }
+
+    acc[row.release] = {
+      ...acc[row.release],
+      [type]: (acc[row.release]?.[type] ?? 0) + (row['avg(span.self_time)'] ?? 0),
+    };
+
+    return acc;
+  }, {});
+}
+
+function SystemApplicationBreakdown({additionalFilters}) {
+  const pageFilter = usePageFilters();
+  const location = useLocation();
+  const {query: locationQuery} = location;
+
+  const {
+    primaryRelease,
+    secondaryRelease,
+    isLoading: isReleasesLoading,
+  } = useReleaseSelection();
+
+  const query = new MutableSearch([
+    'span.op:[app.start.warm,app.start.cold,contentprovider.load,application.load,activity.load]',
+    '!span.description:"Cold Start"',
+    '!span.description:"Warm Start"',
+    ...(additionalFilters ?? []),
+  ]);
+
+  const searchQuery = decodeScalar(locationQuery.query, '');
+  if (searchQuery) {
+    query.addStringFilter(prepareQueryForLandingPage(searchQuery, false));
+  }
+
+  const queryString = appendReleaseFilters(query, primaryRelease, secondaryRelease);
+
+  const {data, isLoading} = useTableQuery({
+    eventView: EventView.fromNewQueryWithPageFilters(
+      {
+        name: '',
+        fields: ['release', 'span.op', 'span.description', 'avg(span.self_time)'],
```

Comment:
```suggestion
        fields: ['release', 'span.op', 'span.description', 'sum(span.self_time)'],
```

probably the better thing to do?

Reply:
Good callout, thanks! I updated everything to sum: https://github.com/getsentry/sentry/pull/63245/commits/c5f374e0b5c7c9b3b36827cc6249a659c32b265e